website/reverse.php

```diff
@@ -46,7 +46,7 @@
         }
     }
 } elseif ($sOutputFormat != 'html') {
-    userError('Need coordinates or OSM object to lookup.');
+    userError('Need coordinates or OSM object to lookup.', $oParams->getString('format'));
```

Comment:
You must not get the format from the parameters directly like that because it disregards the default setting when the parameter is not given. Use $sOutputFormat instead. This applies to all the appearances in this PR. This might be the issue with your failing test.